quint/io-cli-tests.md

```diff
@@ -283,23 +283,23 @@ quint run --init=Init --step=Next --seed=abcde --max-steps=4 \
 An example execution:
 ---------------------------------------------
 action step0 = all {
-  counters::n' = 1,
+  n' = 1,
```

Comment:
This works because we only need to evaluate the main module now that it is self-contained by flattening. WDYT @konnov ? Is there something I'm missing here or is this great? :sweat_smile:

Reply:
Yes, these traces look much better now! I was also annoyed by `counters::`.